pom.xml

```diff
@@ -37,6 +37,14 @@
     <description>An httpd-based load balancer</description>
     <url>http://jboss.org/mod_cluster</url>
 
+    <licenses>
+        <license>
+            <name>GNU Lesser General Public License v3.0 or any later version</name>
```

Comment:
Can you please change to "GNU Lesser General Public License v3.0 or later" so it conforms to the name from list https://spdx.org/licenses/ ?

Reply:
I don't think that conforms to the wording in http://repository.jboss.org/licenses/lgpl-3.0.txt

```
  6. Revised Versions of the GNU Lesser General Public License.

  The Free Software Foundation may publish revised and/or new versions
of the GNU Lesser General Public License from time to time. Such new
versions will be similar in spirit to the present version, but may
differ in detail to address new problems or concerns.

  Each version is given a distinguishing version number. If the
Library as you received it specifies that a certain numbered version
of the GNU Lesser General Public License "or any later version"
applies to it, you have the option of following the terms and
conditions either of that published version or of any later version
published by the Free Software Foundation. If the Library as you
received it does not specify a version number of the GNU Lesser
General Public License, you may choose any version of the GNU Lesser
General Public License ever published by the Free Software Foundation.
```

Reply:
I guess that's fine. Let me change it then.